app/scripts/views/sign_up.js

```diff
@@ -178,6 +178,10 @@ define(function (require, exports, module) {
         showSyncSuggestion: this.isSyncSuggestionEnabled()
       };
 
+      var isFirefox = typeof InstallTrigger !== 'undefined';
```

Comment:
Hey @vershwal do not use ` typeof InstallTrigger !== 'undefined'`, https://github.com/mozilla/fxa-content-server/blob/5bca42f6b2c9d265199552229d5edee614119c5d/app/scripts/lib/user-agent.js#L48